scripts/pc2sandbox.sh

```diff
@@ -158,7 +212,12 @@ DEBUG echo ...done.
 if test -d $PC2_SANDBOX_CGROUP_PATH
 then
 	DEBUG echo Removing existing sandbox to start clean
-	rmdir $PC2_SANDBOX_CGROUP_PATH
+	KillcgroupProcs
+	if ! rmdir $PC2_SANDBOX_CGROUP_PATH
+	then
+		DEBUG echo Cannot remove previous sandbox: $PC2_SANDBOX_CGROUP_PATH
+		exit $FAIL_INVALID_CGROUP_INSTALLATION
```

Comment:
Is this really an "Invalid CGroup Installation" issue?  It seems to me that this is more of a NEW condition introduced by the updates of the PR... in which case I would think a separate error/exit code indicating more precisely the condition for the termination would be better.

Reply:
How about "FAIL_SANDBOX_ERROR" which is one of the errors?  That error is used if the PID can't be added to a cgroup,  Since the case we are talking about deals with an opposite problem (namely, can't remove the cgroup because there are processes in it, presumably), it indicates some sort of general "sandbox error".